frontend/src/metabase/admin/settings/components/SettingsPages/UpdatesSettingsPage.tsx

```diff
@@ -0,0 +1,74 @@
+import cx from "classnames";
+import { c, t } from "ttag";
+
+import { UpsellHostingUpdates } from "metabase/admin/upsells";
+import { useSetting } from "metabase/common/hooks";
+import CS from "metabase/css/core/index.css";
+import { Box, Flex } from "metabase/ui";
+
+import { AdminSettingInput } from "../widgets/AdminSettingInput";
+import { VersionUpdateNotice } from "../widgets/VersionUpdateNotice";
+
+export function UpdatesSettingsPage() {
+  const isHosted = useSetting("is-hosted?");
+  const checkForUpdates = useSetting("check-for-updates");
+  return (
+    <Flex justify="space-between" data-testid="settings-updates">
+      <div style={{ width: "585px" }}>
+        {!isHosted && (
+          <ul>
+            <Box p="0.5rem 1rem 2rem">
+              <AdminSettingInput
+                name="check-for-updates"
+                title={t`Check for updates`}
+                description={t`Identify when new versions of Metabase are available.`}
```

Comment:
this is unnecessary now, I updated the AdminSettingInput to pull in the description from the backend